resource_control: add runaway queries (#13929)
pingcap/docs: mysql-schema.md

## TTL related system tables

* `tidb_ttl_table_status`: the previously executed TTL job and ongoing TTL job for all TTL tables
* `tidb_ttl_task`: the current ongoing TTL subtasks
* `tidb_ttl_job_history`: the execution history of TTL tasks in the last 90 days

## Runaway queries related system tables

> * `tidb_runaway_queries`: the history records of all identified runaway queries over a certain period of time

Author: ```suggestion
* `tidb_runaway_queries`: the history records of all identified runaway queries over the past 7 days
```

> * `tidb_runaway_quarantined_watch`: Quarantined watch rules for runaway queries now in effect

Author: ```suggestion
* `tidb_runaway_quarantined_watch`: quarantined watch rules in effect for runaway queries
```

> * `tidb_runaway_quarantined_watch`: the quarantined watch rules in effect for runaway queries

Author: ```suggestion
* `tidb_runaway_quarantined_watch`: the in effect and outdated quarantined watch rules for runaway queries
```

> * `tidb_runaway_quarantined_watch`: the in effect and outdated quarantined watch rules for runaway queries

Author: ```suggestion
* `tidb_runaway_quarantined_watch`: most recently active quarantined watch rules for runaway queries (contains both currently active rules and recently expired rules)
```

> * `tidb_runaway_quarantined_watch`: most recently active quarantined watch rules for runaway queries (contains currently valid rules, and possibly also contains recently expired rules)

Reviewer: ```suggestion
* `tidb_runaway_quarantined_watch`: the most recently active quarantined watch rules for runaway queries (contains currently valid rules, and possibly also contains recently expired rules)
```

> * `tidb_runaway_quarantined_watch`: the most recently active quarantined watch rules for runaway queries (contains currently valid rules, and possibly also contains recently expired rules)

Author: ```suggestion
* `tidb_runaway_quarantined_watch`: the most recently active quick identification rules for runaway queries (contains currently valid rules, and possibly also contains recently expired rules)
```
